2-0-data-structures-and-algorithms/src/test/java/com/bobocode/array_list/ArrayListTest.java

```diff
@@ -6,51 +6,46 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestMethodOrder;
 
+import java.lang.reflect.Field;
 import java.util.NoSuchElementException;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
-public class ArrayListTest {
+class ArrayListTest {
 
     private List<Integer> arrayList = new ArrayList<>();
 
     @Test
     @Order(1)
-    public void add() {
+    void add() {
         arrayList.add(10);
         arrayList.add(15);
         arrayList.add(20);
 
-        assertThat(arrayList.get(0)).isEqualTo(10);
-        assertThat(arrayList.get(1)).isEqualTo(15);
-        assertThat(arrayList.get(2)).isEqualTo(20);
+        assertThat(getTestArray()[0]).isEqualTo(10);
```

Comment:
@erneman1 I guess it will be clearer when we first get the array and then do all assertions. Since calling a method and adding square brackets right away does not look good. So let's separate that in all tests. 
E.g. 

```java
Object[] internalArray = getTestArray();

assertThat(internalArray[0]).isEqualTo(10);
```